tasks/delete.yml

```
@@ -36,8 +36,9 @@
         state: absent
       notify:
         - reload systemd
-  when:
-    - ansible_lsb.release is version('20.04', '<')
+  when: |
+    ansible_lsb.release is defined and
+    ansible_lsb.release is version('20.04', '<')
```

```suggestion
  when:
    - ansible_lsb.release is defined
    - ansible_lsb.release is version('20.04', '<')
```
If you use lists for `when` then they're `AND` by default anyway.